templates/js/custom-copilot-assistant-new/src/app/app.js.tpl

```diff
@@ -22,6 +22,9 @@ const model = new OpenAIModel({
 
   useSystemMessages: true,
   logRequests: true,
+  {{#CEAEnabled}} 
+  stream: true,
```

Comment:
current teams-ai version is 1.1.0 in package.json. Does this version support stream?

Reply:
Agree we can upgrade the teams-ai version in package.json BTW

Reply:
in package-lock.json, the dependencies is "@microsoft/teams-ai": "^1.1.0", it allows changes that do not modify the left-most non-zero digit in the [major, minor, patch] tuple
so in fact, npm will down teams-ai 1.5.3

Reply:
Yes I know, but let's upgrade it even if we use ^